tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -609,15 +758,15 @@ def cluster_template_create_edit(userToken):
     cluster_cleanup(client, cluster)
 
 
-def node_template_do(userclient):
+def node_template_do(userclient, nodesize):
```

Comment:
Question here, there's an existing `node_template_do` fixture in `test_rke_cluster_provisioning`. Are we sure these names won't collide at run time if we execute `test_rke_cluster_provisioning` and `test_cluster_templates` with pytest?

Reply:
Should not have any effect. Renamed the method ...